rules/sinks/leakages/logs/go.yaml

```diff
@@ -0,0 +1,37 @@
+sinks:
+
+  - id: Leakages.Log.Error
+    name: Log Error
+    patterns:
+      - "(?i).*(?:log).*[.](error|WithError)(f?)"
```

Comment:
Are we not getting package information in methodFullName?

Reply:
We are getting package name too but above rules are combine all the cases, below are some cases:
1. `<unknown>.<log>.Logger.Error`
2. `github.com/unitedwardrobe/go-log/v2.Logger.WithError`
3. `github.com/sirupsen/logrus.FieldLogger.Errorf`

I have added pattern `[.]` just before call name (like Info, Error, Debug) so that it wont match any unnecessary pattern.

Reply:
Actually there way to tag by methodFullName and `log.error` is different. In earlier case we use query like `cpg.call.methodFullName` in latter we need to check on both variable name `log` kind of variable and the function call `info, error` which require a comparatively different tagging strategy. 

For all frontends we have used the methodFullName strategy which is a preferred one I would say. So would recommend to follow similarly and the latter can result in FP's